actions/git_delete_branch.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: git_delete_branch
```

Comment:
For manually deleting branches that were created when I'm testing the workflows.